concepts/pointers/introduction.md

```diff
@@ -0,0 +1,131 @@
+3. Pointers can also point to a null value, to indicate, that they currently do not point to any object.
+
+## General Syntax
+
+A pointer declaration in C++ involves specifying the data type to which the the pointer is pointing, followed by an asterisk (`*`) and the pointer's name.
+When pointers are declared, they are not automatically initialized.
+Without explicit assignment, a pointer typically holds an indeterminate value, often referred to as a "garbage address."
+While certain compilers might initialize pointers to `nullptr`, this behavior is not guaranteed across all compilers, so it's essential not to rely on it.
+It's best practice to explicitly initialize raw pointers and verify their non-null status before utilization to avoid potential issues.
+
+```cpp
+int* ptr{nullptr}; // Declares a pointer and makes sure it is not invalid
+```
+
+To assign the address of a variable to a pointer, you use the address-of operator (`&`).
+Dereferencing a pointer is done using the _indirection operator_ (`*`) operator.
+
+```cpp
+std::string opponent{"Solomon Lane"};
+// 'ethan' points to the address of the string opponent
+std::string* ethan{&opponent}; 
+// Instead of ethan's, the opponent's name address is given to the passPort
+std::string passportName{*ethan};
+```
+
+Attention: dereferencing has to be done explicitly, while _references_ just worked like an alias.
+
+## Pointer Arithmetic
+
+_Pointer arithmetic_ allows you to perform arithmetic operations on pointers, which is particularly useful when working with arrays.
+Adding an integer to a pointer makes it point to a different element.
+
+```cpp
+// Stargate addresses
+int gateAddresses[] = {462, 753, 218, 611, 977};
+// 'ptr' points to the first element of 'gateAddresses'
+int* ptr{gateAddresses}; 
+// Accesses the third Stargate address through pointer arithmetic
+int dialedAddress{*(ptr + 2)}; 
+// Chevron encoded! Dialing Stargate address:
+openStarGate(dialedAddress);
+```
+
+~~~~exercism/caution
+Pointer arithmetic in C++ can easily lead to __undefined behavior__ if not handled carefully.
+Undefined behavior can manifest in unexpected program outcomes, crashes, or even security vulnerabilities.
+One infamous example of the consequences of undefined behavior occurred in the [explosion of the Ariane 5 rocket][ariane-flight-v88] in 1996, where a software exception caused by the conversion of a 64-bit floating-point number to a 16-bit signed integer led to a catastrophic failure.
+~~~~
+
+## Dynamic Memory Allocation and Null Pointers
+
+In C++, `new` is used to dynamically allocate memory for objects on the heap during runtime.
+It returns a pointer to the allocated memory address, allowing objects to have lifetimes not tied to a specific scope.
+However, it's important to always pair `new` with `delete` to properly deallocate memory.
+Dynamic memory allocation should be used cautiously to prevent common issues like memory leaks and accessing deallocated memory.
```

Comment:
I would even use stronger language to discourage the use of "naked" or "raw" `new`.